README.md

```diff
@@ -119,3 +119,13 @@ New Merged MRs:
 * https://gitlab-master.example.com/myproject/-/merge_requests/109 tests: Add commit c.
     master -> master-79e5 (master)
 To ssh://git@gitlab.example.com:12051/myproject.git
+```
+
+## Development
+### Run the tests
+Integration tests require a submodule at `unit_tests/gerritlab_tests`, with a
```

Comment:
Thanks for adding the doc! I should change the test folder name to just `tests`.